bin/crew

```diff
@@ -828,6 +831,20 @@ def strip_find_files (find_cmd, strip_option = "")
   system "#{find_cmd} | xargs -r sh -c 'for i in \"$0\" \"$@\"; do case \"$(head -c 4 $i)\" in ?ELF|\!?ar) echo \"$i\";; esac ; done' | xargs -r llvm-strip #{strip_option}"
 end
 
+def strip_dir (dir)
+  Dir.chdir dir do
+    # Strip libraries with -S
+    puts "Stripping libraries..."
+    strip_find_files "find . -type f  \\( -name 'lib*.a' -o -name 'lib*.so*' \\) -print", "-S"
+
+    # Strip binaries but not compressed archives
+    puts "Stripping binaries..."
+    extensions = [ 'bz2', 'gz', 'lha', 'lz', 'rar', 'tar', 'tbz', 'tgz', 'txz', 'xz', 'zip' ]
+    inames = extensions.join(' ! -iname *\.')
+    strip_find_files "find . -type f ! -iname *\.#{inames} -perm /111 -print | sed -e '/lib.*\.a$/d' -e '/lib.*\.so/d'"
```

Comment:
Too fast. :)  Was this tested?

Reply:
Tested insofar as it is not throwing any errors:

```
-- Installing: /usr/local/tmp/crew/dest/usr/local/share/vulkan/registry/spec_tools
-- Installing: /usr/local/tmp/crew/dest/usr/local/share/vulkan/registry/spec_tools/util.py
-- Installing: /usr/local/tmp/crew/dest/usr/local/share/vulkan/registry/reg.py
Rename all *.la_tmp files back to *.la
Checking...
Checking for conflicts with files from installed packages...
Stripping libraries...
Stripping binaries...
Archiving...
vulkan_headers is built!
```

Reply:
Also, did you catch @saltedcoffii's suggested additional extensions?